src/Our.Umbraco.Ditto/Ditto.cs

```diff
@@ -135,5 +135,15 @@ public static void RegisterDefaultProcessorType<TProcessorAttributeType>()
         {
             TypeDescriptor.AddAttributes(typeof(TObjectType), new TypeConverterAttribute(typeof(TConverterType)));
         }
+
+        /// <summary>
+        /// Registers a global umbraco application context accessor.
+        /// </summary>
+        /// <typeparam name="TUmbracoApplicationContextAccessorType">The type of the accessor.</typeparam>
+        public static void RegisterUmbracoApplicationContextAccessor<TUmbracoApplicationContextAccessorType>()
+            where TUmbracoApplicationContextAccessorType : IUmbracoApplicationContextAccessor, new()
+        {
+            DittoProcessorRegistry.Instance.RegisterDefaultUmbracoApplicationContextAccessorType<TUmbracoApplicationContextAccessorType>();
```

Comment:
doh! looks like we do have RegisterDefaultProcessorType though, hmmm. Lets keep it how you've got it, but just move it to Ditto including the private property. :)